praetorian_cli/ui/aegis/commands/cp.py

```
@@ -0,0 +1,132 @@
+            user=user,
+            ssh_options=ssh_options,
+            display_info=True,
+            use_rsync=use_rsync,
+        )
+    except Exception as e:
+        colors = getattr(menu, 'colors', DEFAULT_COLORS)
+        menu.console.print(f"[{colors['error']}]Copy error: {e}[/{colors['error']}]")
+        menu.pause()
+
+
+def complete(menu, text, tokens):
```

Ruff (0.15.1)

[warning] Unused function argument: `menu`

(ARG001)